packages/bun-uws/src/HttpContext.h

```diff
@@ -293,6 +295,10 @@ struct HttpContext {
 
             /* We need to uncork in all cases, except for nullptr (closed socket, or upgraded socket) */
             if (returnedSocket != nullptr) {
+                us_socket_t* returnedSocketPtr = (us_socket_t*) returnedSocket; 
+                /* We don't want open sockets to keep the event loop alive between HTTP requests */
+                us_socket_unref(returnedSocketPtr);
+
```

Comment:
@gvilums Is this something we want to push back upstream to `uNetworking/uWebSockets` and `uNetworking/uSockets`?

Example references:
https://github.com/uNetworking/uWebSockets/blob/master/src/HttpContext.h#L269-L272 
https://github.com/uNetworking/uSockets/blob/master/src/socket.c#L193-L231

Reply:
No, this is specific to how our event loop works: In our http server we keep a separate reference into the loop to keep it alive, which is why we need to unref the socket here to shut the server down. This does not make sense in general though as it would lead to the event loop randomly exiting without the server present to reference it.